src/Bookings.js

```diff
@@ -1,8 +1,14 @@
 class Bookings {
```

Comment:
It looks like you captured this in updating `constructor(bookings)` to `constructor(booking)` but it's generally considered best practice for javascript class names to be singular, so `class Booking`, especially here since you're instantiating a single booking object.